mscore/stringutils.cpp

```diff
@@ -0,0 +1,90 @@
+//=============================================================================
+//  MuseScore
+//  Linux Music Score Editor
+//  Copyright (C) 2002-2018 Werner Schweer and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#include "stringutils.h"
+
+namespace Ms {
+
+//---------------------------------------------------------
+//   removeLigatures
+//---------------------------------------------------------
+
+QString stringutils::removeLigatures(const QString& pre)
+      {
+      QString result = pre;
+
+      // Latin Big Letter AA (&#42802) and Latin Small Letter aa (&#42803)
+      result = result.replace(QRegularExpression("[\\x{A732}]"), QString("AA"));
+      result = result.replace(QRegularExpression("[\\x{A733}]"), QString("aa"));
+
+      // Latin Big Letter AE (&#198) and Latin Small Letter ae (&#230), A with above dots, a with above dots
+      result = result.replace(QRegularExpression("[\\x{00C6}\\x{00C4}]"), QString("AE"));
+      result = result.replace(QRegularExpression("[\\x{00E6}\\x{00E4}]"), QString("ae"));
+
+      // Latin Big Letter AO (&#42804) and Latin Small Letter ao (&#42805)
+      result = result.replace(QRegularExpression("[\\x{A734}]"), QString("AO"));
+      result = result.replace(QRegularExpression("[\\x{A735}]"), QString("ao"));
+
+      // Latin Big Letter AU (&#42806) and Latin Small Letter au (&#42807)
+      result = result.replace(QRegularExpression("[\\x{A736}]"), QString("AU"));
+      result = result.replace(QRegularExpression("[\\x{A737}]"), QString("au"));
+
+      // IJ (&#306) and ij (&#307)
+      result = result.replace(QRegularExpression("[\\x{0132}]"), QString("IJ"));
+      result = result.replace(QRegularExpression("[\\x{0133}]"), QString("ij"));
+
+      // Eszett (&#223)
+      result = result.replace(QRegularExpression("[\\x{00DF}]"), QString("ss"));
```

Comment:
As of latest there is also an upper case `ẞ` , &#7838; (compare `ß`), to be used in upper case words, so there it should translate to `SS`